File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/DetailPaInfoFormGridInsureInfo.java
package nta.med.data.model.ihis.system;

import java.sql.Timestamp;

public class DetailPaInfoFormGridInsureInfo {
	private Timestamp startDate;
    private String typeName;
    private Timestamp endDate;
    private String johapName;
    private String percentage;
    private String percentageNo;
    private Timestamp lastCheckDate;
	public DetailPaInfoFormGridInsureInfo(Timestamp startDate, String typeName,
			Timestamp endDate, String johapName, String percentage,
			String percentageNo, Timestamp lastCheckDate) {
		super();
		this.startDate = startDate;
		this.typeName = typeName;
		this.endDate = endDate;
		this.johapName = johapName;
		this.percentage = percentage;
		this.percentageNo = percentageNo;
		this.lastCheckDate = lastCheckDate;
	}
	public Timestamp getStartDate() {
		return startDate;
	}
	public void setStartDate(Timestamp startDate) {
		this.startDate = startDate;
	}
	public String getTypeName() {
		return typeName;
	}
	public void setTypeName(String typeName) {
		this.typeName = typeName;
	}
	public Timestamp getEndDate() {
		return endDate;
	}
	public void setEndDate(Timestamp endDate) {
		this.endDate = endDate;
	}
	public String getJohapName() {
		return johapName;
	}
	public void setJohapName(String johapName) {
		this.johapName = johapName;
	}
	public String getPercentage() {
		return percentage;
	}
	public void setPercentage(String percentage) {
		this.percentage = percentage;
	}
	public String getPercentageNo() {
		return percentageNo;
	}
	public void setPercentageNo(String percentageNo) {
		this.percentageNo = percentageNo;
	}
	public Timestamp getLastCheckDate() {
		return lastCheckDate;
	}
	public void setLastCheckDate(Timestamp lastCheckDate) {
		this.lastCheckDate = lastCheckDate;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/DetailPaInfoFormGridInsureInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
    public class DetailPaInfoFormGridInsureInfo
    {
        private String _startDate;
        private String _typeName;
        private String _endDate;
        private String _johapName;
        private String _percentage;
        private String _percentageNo;
        private String _lastCheckDate;

        public String StartDate
        {
            get { return this._startDate; }
            set { this._startDate = value; }
        }

        public String TypeName
        {
            get { return this._typeName; }
            set { this._typeName = value; }
        }

        public String EndDate
        {
            get { return this._endDate; }
            set { this._endDate = value; }
        }

        public String JohapName
        {
            get { return this._johapName; }
            set { this._johapName = value; }
        }

        public String Percentage
        {
            get { return this._percentage; }
            set { this._percentage = value; }
        }

        public String PercentageNo
        {
            get { return this._percentageNo; }
            set { this._percentageNo = value; }
        }

        public String LastCheckDate
        {
            get { return this._lastCheckDate; }
            set { this._lastCheckDate = value; }
        }

        public DetailPaInfoFormGridInsureInfo() { }

        public DetailPaInfoFormGridInsureInfo(String startDate, String typeName, String endDate, String johapName, String percentage, String percentageNo, String lastCheckDate)
        {
            this._startDate = startDate;
            this._typeName = typeName;
            this._endDate = endDate;
            this._johapName = johapName;
            this._percentage = percentage;
            this._percentageNo = percentageNo;
            this._lastCheckDate = lastCheckDate;
        }
    }

}
